warehouse/templates/manage/token.html

```diff
@@ -61,8 +60,40 @@ <h2>Token for "{{ macaroon.description }}"</h2>
 
       {{ confirm_modal(title=title, confirm_name="Username", confirm_string=user.username, confirm_button_label=confirm_button_label, slug=slug, extra_fields=extra_fields, custom_warning_text=token_warning_text, confirm_string_in_title=False) }}
     </div>
-    <hr>
+
+    <h2>Using this token</h2>
+
+    <p>To use this API token:</p>
+
+    <ul>
+      <li>Set your username to <code>__token__</code></li>
+      <li>Set your password to the token value, including the <code>pypi-</code> prefix</li>
+    </ul>
+
+    {% if macaroon.caveats.permissions == "user" %}
+    <p>For example, if you are using <a href="https://pypi.org/project/virtualenv/">Twine</a> to upload your projects to PyPI, set up your <code>$HOME/.pypirc</code> file like this:</p>
+<pre class="code-block">
+[pypi]
+  username = __token__
+  password = {{ serialized_macaroon }}
+</pre>
+    {% else %}
+    <p>For example, if you are using <a href="https://pypi.org/project/virtualenv/">Twine</a> to upload multiple projects to PyPI, you can set up a <code>$HOME/.pypirc</code> file like this:</p>
+<pre class="code-block">
```

Comment:
Indentint this and the closing line 4 spaces fixed the linting issue for me, I *think* it still matches visually. You may want to try that?

Reply:
hmmm, that's not working for me. When I intent, I am getting:

![Screenshot from 2019-09-15 12-18-20](https://user-images.githubusercontent.com/3323703/64921113-e1926c00-d7b6-11e9-9f20-92d85084fd79.png)